server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java

```diff
@@ -139,6 +139,11 @@ public CeMainThread() {
     public void run() {
       boolean startupSuccessful = attemptStartup();
       this.started = true;
+      try {
+        Thread.sleep(2000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1148)